projects/js-packages/components/components/record-meter-bar/index.tsx

```diff
@@ -106,6 +107,23 @@ const RecordMeterBar: React.FC< RecordMeterBarProps > = ( {
 					} ) }
 				</ul>
 			</div>
+			<table className="screen-reader-text">
+				<caption>{ __( 'Summary of the records', 'jetpack' ) }</caption>
+				<tbody>
+					<tr>
+						<th scope="col">{ __( 'Record type', 'jetpack' ) }</th>
+						<th scope="col">{ __( 'Record count', 'jetpack' ) }</th>
+					</tr>
+					{ itemsToRender.map( ( { label, count } ) => {
+						return (
+							<tr key={ label }>
+								<td key={ label }>{ label }</td>
+								<td key={ count }>{ count }</td>
```

Comment:
Looks like the key is only needed on the top level element of each list member (the `<tr>`). It would probably be best to remove them from the `<td>`s to avoid confusion. Particularly since the "count" as a key seems reasonably likely to not actually be unique, which would break things if it were actually used.
```suggestion
								<td>{ label }</td>
								<td>{ count }</td>
```
P.S. If it's possible that multiple rows might have the same label, that might need some further thought to avoid confusing react. But that would probably be a matter for a separate PR, since that would affect linens 78 and 86 above too.

Reply:
Thanks @anomiex 👍 you're right. 

>P.S. If it's possible that multiple rows might have the same label, that might need some further thought to avoid confusing react. But that would probably be a matter for a separate PR, since that would affect lines 78 and 86 above too.

good point. I don't think it's possible that multiple rows would have the same label, but if it were to be the case it would certainly be cause for a separate PR as I suspect it would cause a number of other problems around the place too.